scripts/generators/foundry.py

```diff
@@ -1,20 +1,21 @@
-import yaml
 from glob import glob
 import json
 import os
-import pandas as pd
-import re
 from shutil import copy
 import fire
-import time
 import datetime
 
-secrets = json.load(open('../../secrets.json'))
+# Moves with fields that are problematic for the export
+# - Non-moves such as "Run Away" are excluded
+# - Struggle and Growl have attributes like "Strength/special"
+# - Copycat Damage1 is "Same as the copied move"
```

Comment:
No idea how to make that work with Foundry, but it is correct

Reply:
By leaving the attribute empty and adding a note to Added Effects probably